.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,81 @@
+            <item index="14" class="java.lang.String" itemvalue="greenlet" />
+            <item index="15" class="java.lang.String" itemvalue="six" />
+            <item index="16" class="java.lang.String" itemvalue="botocore" />
+            <item index="17" class="java.lang.String" itemvalue="boto" />
+            <item index="18" class="java.lang.String" itemvalue="mypy-extensions" />
+            <item index="19" class="java.lang.String" itemvalue="marshmallow" />
+            <item index="20" class="java.lang.String" itemvalue="packaging" />
+            <item index="21" class="java.lang.String" itemvalue="APScheduler" />
+            <item index="22" class="java.lang.String" itemvalue="dataclasses-json" />
+            <item index="23" class="java.lang.String" itemvalue="schedule" />
+            <item index="24" class="java.lang.String" itemvalue="boto3" />
+            <item index="25" class="java.lang.String" itemvalue="typing-inspect" />
+            <item index="26" class="java.lang.String" itemvalue="Flask-Mail" />
+            <item index="27" class="java.lang.String" itemvalue="marshmallow-enum" />
+            <item index="28" class="java.lang.String" itemvalue="zipp" />
+            <item index="29" class="java.lang.String" itemvalue="typing_extensions" />
+            <item index="30" class="java.lang.String" itemvalue="charset-normalizer" />
+            <item index="31" class="java.lang.String" itemvalue="pycryptodome" />
+            <item index="32" class="java.lang.String" itemvalue="urllib3" />
+            <item index="33" class="java.lang.String" itemvalue="uritemplate" />
+            <item index="34" class="java.lang.String" itemvalue="google-auth-httplib2" />
+            <item index="35" class="java.lang.String" itemvalue="httplib2" />
+            <item index="36" class="java.lang.String" itemvalue="google-api-python-client" />
+            <item index="37" class="java.lang.String" itemvalue="pyparsing" />
+            <item index="38" class="java.lang.String" itemvalue="idna" />
+            <item index="39" class="java.lang.String" itemvalue="requests-oauthlib" />
+            <item index="40" class="java.lang.String" itemvalue="google-auth-oauthlib" />
+            <item index="41" class="java.lang.String" itemvalue="oauthlib" />
+            <item index="42" class="java.lang.String" itemvalue="certifi" />
+            <item index="43" class="java.lang.String" itemvalue="websockets" />
+            <item index="44" class="java.lang.String" itemvalue="blinker" />
+            <item index="45" class="java.lang.String" itemvalue="sentry-sdk" />
+            <item index="46" class="java.lang.String" itemvalue="phonepe-sdk" />
+          </list>
+        </value>
+      </option>
+    </inspection_tool>
+    <inspection_tool class="PyPep8NamingInspection" enabled="true" level="WEAK WARNING" enabled_by_default="true">
+      <option name="ignoredErrors">
+        <list>
+          <option value="N806" />
```

Comment:
The values 'N806' and 'N802' in the ignoredErrors list should be properly formatted. They should be 'N806: variable in function should be lowercase' and 'N802: function name should be lowercase' respectively.